code/modules/telesci/telesci_computer.dm

```diff
@@ -301,6 +309,21 @@
 			flick(transition_icon, A)
 	qdel(src)
 
+/obj/effect/teleport_blink/container_resist(mob/living/M)
+	if(!reset_turf)
+		return
+	if(prob(50))
+		to_chat(M, "<span class='warning'>You resist the teleportation!</span>")
+		visible_message("<span class='warning'>The teleport beam collapses!</span>")
+		M.forceMove(reset_turf)
+		M.adjustToxLoss(15) // A bit of toxins damage.
```

Comment:
But why does it deal 15 toxin damage? It just makes no sense...

Also, you can do that to 20 different crewmembers who don't turn their suit sensors off. A pretty bad idea considering they most likely won't be able to tell where they were being teleported to, so you'll leave no trace.